stable/kommander/requirements.lock

```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.1.4
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: 0.3.4
+  version: v0.3.5
```

Comment:
```suggestion
  version: 0.3.5
```

Reply:
well that is what `helm dependency update` did :P 
and it is correct https://github.com/mesosphere/kommander-cluster-lifecycle/releases